README.md

```diff
@@ -48,7 +48,130 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 )
 ```
 
-## Setting up the accessibility test folder
+## Plugin Setup
+
+There are two different approaches to writing accessibility tests, you may choose the approach that
+suites your team best.
+
+## Automatic accessibility testing 
+
+Automatic accessibility testing allows you to write tests via a single spec, it can do this by
+automatically scanning through your project for views.
+
+### Example Accessibility Spec
+
+```scala
+import config.AppConfig
+import controllers.{ContactForm, ContactHmrcForm}
+import model.{AccessibilityForm, FeedbackForm, ReportProblemForm, SurveyForm}
+import org.scalacheck.Arbitrary
+import org.scalatest.Reporter
+import org.scalatest.events.{Event, SuiteCompleted, TestPending}
+import play.api.data.Form
+import play.api.data.Forms._
+import play.twirl.api.Html
+import uk.gov.hmrc.scalatestaccessibilitylinter.views.AutomaticAccessibilitySpec
+import views.html._
+import scala.collection.mutable.ListBuffer
+
+// this is the class that any consuming team would have to implement
+class FrontendAccessibilitySpec
+  extends AutomaticAccessibilitySpec {
+
+  // these are things that need to have sane values for pages to render properly - there may be others
+  /** example
+      val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
+      implicit val arbConfig: Arbitrary[AppConfig] = fixed(appConfig)
+  */
+
+  // we can currently generate Arbitrary[T] but not Arbitrary[T[_]], so service teams need to provide these explicitly.
+  // however, these should generally be available in the existing service codebase.
+  // these are implicit to simplify calls to render() below
+  // e.g implicit val arbReportProblemPage: Arbitrary[Form[ReportProblemForm]] = fixed(reportProblemForm)
+
+  // this is the package where the views live in the service, you may need to update this path
+  // as your project might have views in a different location
+  val viewPackageName = "views.html"
+
+  // this is the layout class which is injected into all full pages in the service
+  // and may be different in your service
+  val layoutClasses = Seq(classOf[views.html.components.Layout])
+
+  // this partial function wires up the generic render() functions with arbitrary instances of the correct types.
+  // IntelliJ invariably shows some or all of these as red :(
```

Comment:
what about, "Important: there's a known issue with intellij incorrectly displaying warnings here, you should be able to ignore these"